src/main/resources/application.properties

```diff
@@ -54,3 +54,19 @@ json.max.size.limit=2000000
 # Use false for Deloitte sandbox
 check.http.header=false
 check.media.type.json=false
+
+# Configuration for External Admin JWT Authentication
+# External Admin JWTs must have this role to be accepted
+admin.jwt.role=${ELRR_ADMIN_JWT_ROLE:elrr-admin}
```

Comment:
you should have default values for application-local as well.

Reply:
wait does this roll back to the defaults after the `:` and then carry over to override local when its not set?? That would be cool.